NEWS

```diff
@@ -32,6 +32,8 @@ x/y/2019 ola-0.10.8
    Python API
  * Renamed RESONSE_INVALID_DESTINATION(sic) to RESPONSE_INVALID_DESTINATION in
    the ArduinoWidget code
+ * Fix compatibility with ncurses 6
+ * Fix compatibility with newer versions of Protobuf
```

Comment:
Can you add the version number in here too (3.7 by my reckoning).

Reply:
Can we have the lower window of the versions too please @kripton (i.e. 3.7 to 3.11.4), while the upper bound you've tested with is good to know, being able to check the NEWS file and tell someone they need to be running 0.10.8 to work with Protobuf 3.7 is even more invaluable. Hopefully this fix will still be sufficient for Protobuf 3.12 for example.

Reply:
So actually probably something like "Fix compatibility with Protobuf 3.7 (tested with up to 3.11.4)" should cover it.